lib/__tests__/_sendEvent.test.ts

```diff
@@ -93,13 +100,62 @@ describe("sendEvents", () => {
         }
       ]);
       const requestUrl = XMLHttpRequest.open.mock.calls[0][1];
-      const { query } = url.parse(requestUrl);
-      expect(querystring.parse(query!)).toEqual({
+      const { searchParams } = new URL(requestUrl);
+      expect(searchParamsToObject(searchParams)).toEqual({
         "X-Algolia-API-Key": "testKey",
         "X-Algolia-Agent": "insights-js (1.0.1); insights-js-node-cjs (1.0.1)",
         "X-Algolia-Application-Id": "testId"
       });
     });
+    it("should infer query ids when additionalParams.inferQueryID is true", () => {
```

Comment:
Should we add the inverse? I.e., if `inferQueryID` is false?

Reply:
Yep, I'll add that in a follow-up PR.